### DISCUSSION TITLE
Unable to login to Github with Copilot on IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by edurdevic** [community/community#39827] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I activated Copilot and followed the instructions to authenticate my github account from IntalliJ IDEA.

The authentication is successful on the Github website, but IntelliJ returns this error

![Screenshot 2022-11-23 at 10 49 08](https://user-images.githubusercontent.com/2878017/203517345-bb7b89f1-8065-4af6-b14a-b6a9b9f7c3cf.png)

I am able to authenticate correctly from VS Code.

I am using:
IntelliJ IDEA 2021.2.3 (Community Edition)
Build #IC-212.5457.46, built on October 12, 2021
Runtime version: 11.0.12+7-b1504.40 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.5.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry: scala.erase.compiler.process.jdk.once=false
Non-Bundled Plugins: com.github.copilot (1.1.33.1980), org.intellij.scala (2021.2.30)
Kotlin: 212-1.5.10-release-IJ5457.46

**Comment by ossanna16** [comment]
Hi there @edurdevic and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by huyl** [reply]
How do you know if it's a great question if you don't read it? :stuck_out_tongue:
